js/schema/plugin.json

```diff
@@ -44,18 +44,24 @@
           "type": "string"
         },
         "url": {
-          "description": "A url pointing to an fs resource, as per the PyFilesystem fsurl specification",
+          "description": "A url pointing to an fs resource, as per the PyFilesystem fsurl or fsspec url specification",
           "type": "string",
           "pattern": "^.+?:\/\/([^:]*:.*@.*|[^@]*)$"
         },
+        "type": {
+          "description": "Directory to be first opened (e.g., myDir/mySubdir)",
+          "type": "string",
+          "enum": ["pyfs", "fsspec"],
+          "default": "pyfs"
+        },
         "preferred_dir": {
           "description": "Directory to be first opened (e.g., myDir/mySubdir)",
           "type": "string"
         },
         "auth": {
-          "description": "Given any template {{VARS}} in the url, 'ask' (default) to open a dialog box asking for credentials, or `env` to pick up credentials from the server's environment variables",
+          "description": "Given any template {{VARS}} in the url, 'ask' (default) to open a dialog box asking for credentials, `env` to pick up credentials from the server's environment variables, or `none`.",
           "type": "string",
-          "enum": ["ask", "env", false],
+          "enum": ["ask", "env", "none"],
```

Comment:
I think this will break the existing settings for anyone with this set to `false`, as it will fail the schema validation for those users?

Reply:
weirdly Jupyter never let me choose `false`, i would get an error immediately as it was not a string type. Have you seen this working?

Reply:
You are right, I cannot set this either, so this is a straight bugfix then! 👍